ogr/ogrsf_frmts/openfilegdb/filegdbtable.cpp

```diff
@@ -2547,7 +2548,7 @@ int FileGDBTable::GetIndexCount()
         returnErrorAndCleanupIf(static_cast<GUInt32>(pabyEnd - pabyCur) <
                                     sizeof(GUInt32),
                                 VSIFree(pabyIdx));
-        GUInt32 nIdxNameCarCount = GetUInt32(pabyCur, 0);
+        const GUInt32 nIdxNameCarCount = GetUInt32(pabyCur, 0);
```

Comment:
Was `Car` actually meant to be `Char` ?

Reply:
addressed in https://github.com/OSGeo/gdal/pull/11410